### PR TITLE
Add links to tickets after description paragraph

### PR DESCRIPTION
**Pull request by alanmoo**
Closes https://github.com/MozillaFoundation/2016.Mozillafestival.org/issues/79

### CODE REVIEW

File: pages/tickets.jsx [hunk 1]
```diff
@@ -29,9 +29,9 @@ var Tickets = React.createClass({
           <div ref="titoContainer"></div>
           <div className="ticket-desc">
             <p className="boldish">Youth Ticket £3</p>
-            <p>This ticket covers entry to the festival on Saturday and Sunday. It includes lunch, beverages, swag, and two days of building, making and learning. Youth tickets are for those under 18. Youth under 16 must be accompanied by an adult. If you are an educator bringing young people to the event, please email us at <a href="mailto:festival@mozilla.org">festival@mozilla.org</a> before you buy tickets as we can help with group discounts.</p>
+            <p>This ticket covers entry to the festival on Saturday and Sunday. It includes lunch, beverages, swag, and two days of building, making and learning. Youth tickets are for those under 18. Youth under 16 must be accompanied by an adult. If you are an educator bringing young people to the event, please email us at <a href="mailto:festival@mozilla.org">festival@mozilla.org</a> before you buy tickets as we can help with group discounts. <a href="https://ti.to/Mozilla/mozfest-2016/" target="_blank">Buy a youth ticket</a></p>
```
Comment (mmmavis) [inline pages/tickets.jsx]: let's add a period after "Buy a youth ticket"

File: pages/tickets.jsx [hunk 2]
```diff
@@ -29,9 +29,9 @@ var Tickets = React.createClass({
...
             <p className="boldish">Adult Ticket £45</p>
-            <p>This ticket covers entry to the festival on Friday night, Saturday and Sunday. It includes lunch, coffee, adult beverages, swag, and two days of building, making and learning (plus a party on Saturday evening!) If you are a parent with young children, we have a free creche onsite open Saturday and Sunday.</p>
+            <p>This ticket covers entry to the festival on Friday night, Saturday and Sunday. It includes lunch, coffee, adult beverages, swag, and two days of building, making and learning (plus a party on Saturday evening!) If you are a parent with young children, we have a free creche onsite open Saturday and Sunday. <a href="https://ti.to/Mozilla/mozfest-2016/" target="_blank">Buy an adult ticket</a></p>
```
Comment (mmmavis) [inline pages/tickets.jsx]: let's add a period after "Buy an adult ticket"

**Comment by mmmavis** [comment]
R+ after adding commas 🍻

**Comment by alanmoo** [comment]
Added.

**Pull request merged**